pkg/api/api.go

```diff
@@ -343,10 +344,10 @@ func (hs *HTTPServer) registerRoutes() {
 			alertsRoute.Get("/states-for-dashboard", Wrap(GetAlertStatesForDashboard))
 		})
 
-		apiRoute.Get("/alert-notifications", Wrap(GetAlertNotifications))
-		apiRoute.Get("/alert-notifiers", Wrap(GetAlertNotifiers))
+		apiRoute.Get("/alert-notifiers", reqEditorRole, Wrap(GetAlertNotifiers))
 
 		apiRoute.Group("/alert-notifications", func(alertNotifications routing.RouteRegister) {
+			apiRoute.Get("/", Wrap(GetAlertNotifications))
```

Comment:
not sure but this shouldn't be:
`alertNotifications.Get("/", Wrap(GetAlertNotifications))`?

Reply:
Good catch, will fix :+1: